client/components/layout/cards/poolCard.tsx

```diff
@@ -0,0 +1,42 @@
+import AddToPoolButton from "@/components/buttons/iconButtons/addToPoolButton";
+import DeleteButton from "@/components/buttons/iconButtons/deleteButton";
+import ShowMoreIconButton from "@/components/buttons/iconButtons/showMoreIconButton";
+import { Header3 } from "@/components/textComponents";
+import Album from "@/types/albumTypes";
+import Artist from "@/types/artistTypes";
+import Playlist from "@/types/playlistTypes";
+import Track from "@/types/trackTypes";
+import { Box, Card } from "@mui/material";
+
+export default function PoolCard(props: {
+    poolItem: Track | Album | Playlist | Artist
+    token: string
+    handleDelete: (itemToDelete: Album | Track | Artist | Playlist) => void
+}) {
+
+    const truncatedName = props.poolItem.name.length > 25 ? props.poolItem.name.slice(0, 25) + "..." : props.poolItem.name;
+
+    return (
+        <Card sx={{ bgcolor: 'secondary.light', width: 1, minHeight: 66 }}>
+            <Box sx={{ display: 'flex', alignItems: 'center', justifyContent: 'space-between' }}>
+                <Box sx={{ display: 'flex', alignItems: 'center' }}>
+                    <Box
+                        sx={{
+                            width: 50,
+                            height: 50,
+                            //backgroundImage: `url(${props.poolItem.icon_link})`,
```

Comment:
Should this be used or deleted?

Reply:
Fixed but since a track does not have an image I am going to make some bugfix tickets after this is merged since I think we might have some issues after the merge

Reply:
Track has album which has image, you should be able to use that

Reply:
Yes but this is a fix for another ticket anyway